parm/config/config.base.emc.dyn

```diff
@@ -227,11 +233,77 @@ export lobsdiag_forenkf=".true."
 export DO_GLDAS="YES"
 export gldas_cyc=00
 
-# run wave component
-export DO_WAVE="NO"
+# APP settings
+export APP="ATM"
+
+if [ $APP = "ATM" ]; then
+  export DO_WAVE="NO"
+  export DO_MOM6="NO"
+  export DO_CICE="NO"
+  export DO_GOCART="NO"
+  export CCPP_SUITE="FV3_GFS_v16,FV3_GFS_v16_ugwpv1"
+elif [ $APP = "ATMW" ]; then
+  export DO_WAVE="YES"
+  export DO_MOM6="NO"
+  export DO_CICE="NO"
+  export DO_GOCART="NO"
+  export CCPP_SUITE="FV3_GFS_v16,FV3_GFS_v16_ugwpv1"
+elif [ $APP = "S2S" ]; then
+  export DO_WAVE="NO"
+  export DO_MOM6="YES"
+  export DO_CICE="YES"
+  export DO_GOCART="NO"
+  export CCPP_SUITE="FV3_GFS_v16,FV3_GFS_v16_coupled_nsstNoahmpUGWPv1"
+elif [ $APP = "S2SW" ]; then
+  export DO_WAVE="YES"
+  export DO_MOM6="YES"
+  export DO_CICE="YES"
+  export DO_GOCART="NO"
+  export CCPP_SUITE="FV3_GFS_v16,FV3_GFS_v16_coupled_nsstNoahmpUGWPv1"
+elif [ $APP = "ATMAERO" ]; then
+  export DO_WAVE="NO"
+  export DO_MOM6="NO"
+  export DO_CICE="NO"
+  export DO_GOCART="YES"
+  export CCPP_SUITE="FV3_GFS_v16,FV3_GFS_v16_ugwpv1"
+fi
+
 if [[ "$SDATE" -lt "2019020100" ]]; then DO_WAVE="NO" ; fi # no rtofs in GDA
 export WAVE_CDUMP="both"
 
+# Coupling settings
+export cpl=".true."
```

Comment:
cpl should be .false. until one of the coupling things is turned on.

Reply:
By "one of the coupling things" do you mean the other cpl* variables?

Reply:
How about the adjustment below? What should cplchem be by default?

```
# Coupling settings

export cplflx=".false."
export cplflx=".true."
export cplice=".false."
export cplgocart=".false."
export cplwav=".false."
export cplwav2atm=".false."

if [ $DO_CICE = "YES" ]; then
  export cplice=".true."
fi
if [ $DO_GOCART = "YES" ]; then
  export cplgocart=".true."
fi
if [ $DO_WAVE = "YES" ]; then
  export cplwav=".true."
fi

if [ $cplflx = ".true." -o $cplice = ".true." -o $cplwav = ".true." -o $cplchem = ".true." ]; then
  export cpl=".true."
fi
```

Reply:
I moved the cpl variables into config.fcst.